Source/Initialization/InjectorMomentum.H

```diff
@@ -203,6 +244,13 @@ struct InjectorMomentumRadialExpansion
         return {x*u_over_r, y*u_over_r, z*u_over_r};
     }
 
+    AMREX_GPU_HOST_DEVICE
+    amrex::XDim3
+    getBulkMomentum (amrex::Real x, amrex::Real y, amrex::Real z) const noexcept
+    {
+        return {x*u_over_r, y*u_over_r, z*u_over_r};
```

Comment:
This is correct provided `u_over_r` is in `m**[-1]`, i.e., `u_over_r` is the normalized momentum `gamma*beta` divided by the physical position (in SI). As this is not written anywhere in the code or in the doc, could you add a little doctsring in the `InjectorMomentumRadialExpansion` constructor to explain that? Thanks.